project/Dependencies.scala

```diff
@@ -3,7 +3,8 @@ import sbt._
 object Dependencies {
   private[Dependencies] object V {
     val cats      = "2.2.0"
-    val curator   = "5.3.0"
+    val curator   = "4.2.0"
+    val curatorTest = "2.12.0"
```

Comment:
Why not use the same version  for both `curator-test` and `curator-framework`?

Reply:
There are two versions to pin to keep compatibility with Zk 3.4.X according to the docs: https://curator.apache.org/docs/zk-compatibility-34/.

 Recipes at 4.2.0

> Curator 4.2.x supports ZooKeeper 3.4.x ensembles in a soft-compatibility mode. To use this mode you must exclude ZooKeeper when adding Curator to your dependency management tool.

and test at 2.12.0

> If you wish to use Curator's TestingServer with ZooKeeper 3.4.x you must use the older version of it (in addition to the instructions above)

Reply:
Oh I see. I overlooked that part about testing, sorry.